src/UI/walletConnect/WalletConnectLoginContainer/index.tsx

```diff
@@ -171,11 +172,16 @@ export const WalletConnectLoginContainer = ({
     }
   }, [displayWalletConnectV2]);
 
+  const authorizationInfo = showScamPhishingAlert
+    ? getAuthorizationInfo(token)
+    : undefined;
```

Comment:
it's recommended to use `null` instead of `undefined`, because `null` is a value